src/components/pages/lessons/overlay/recommend-next-step-overlay/lesson-01.js

```diff
@@ -0,0 +1,120 @@
+export const cloudflareLesson = {
```

Comment:
this can be used anywhere you need static data for a video resource (lesson) that is associated with a collection